Prepare for TF review: `video` element visual-only content has accessible alternative (c3232f) (#1404)
act-rules/act-rules.github.io: _rules/video-only-alternative-for-visual-c3232f.md

> #### Passed Example 1

Reviewer: I'm considering whether we should take the opportunity to reorder the examples in "most common to least common" order. I feel that having a video as an alternative for text is like super rare and that transcript or video is more common.

Author: That makes sense. Updating accordingly.

> This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such. (Passes rule [`Video` Element Visual-Only Content Is Media Alternative For Text](https://act-rules.github.io/rules/fd26cf))

Reviewer: I like that way of giving both infos. Maybe even
```suggestion
This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such. Thus, it passes rule [`Video` Element Visual-Only Content Is Media Alternative For Text](https://act-rules.github.io/rules/fd26cf).
```
(avoids the sentence composed of only a parenthesis)

</html>
```

#### Failed Example 2

> This `video` element, which has no audio, has a incorrect text transcript available through a link on the same page.

Reviewer 2: ```suggestion
This `video` element, which has no audio, has an incorrect text transcript available through a link on the same page.
```

</html>
```

#### Failed Example 4

> This `video` element, which has no audio, is a media alternative for the text in the page but not labeled as such.

Reviewer 2: ```suggestion
This `video` element, which has no audio, is a media alternative for the text in the page but it is not labelled as such.
```
